rep-0149.rst

```diff
@@ -346,8 +346,8 @@ Example
     </description>
     <maintainer email="someone@example.com">Someone</maintainer>
 
-    <license>BSD</license>
-    <license file="LICENSE">LGPL</license>
+    <license file="LICENSE" type="spdx">BSD-3-Clause</license>
+    <license source-files="include/my_package/linear_math/*">Zlib</license>
```

Comment:
```suggestion
    <license files="include/my_package/linear_math/*">Zlib</license>
```

Doesn't seem relevant to limit this to _source_ files, any files might be licensed.

Reply:
Since there is already a attribute `file` to specify the path to the license file, I proposed `source-files` to avoid confusion.

Reply:
👍  Ah, I had not noticed that.